### PR TITLE
Partial translate for 01-1

### PR DESCRIPTION
**Pull request by lejinghu**
From "Course plan" to "History of Pattern Recognition and introduction to Gradient Descent".

### CODE REVIEW

**Comment by Atcold** [comment]
Hi @lejinghu. Can you send this pull request to the main repo?
https://github.com/Atcold/pytorch-Deep-Learning

Thanks!

**Comment by lejinghu** [comment]
Done.
